test/fixtures/sourcemap.run.js

```diff
@@ -0,0 +1,12 @@
+const rimraf = require('rimraf')
+const path = require('path')
+const ClinicDoctor = require('../../index.js')
+
+const doctor = new ClinicDoctor()
+doctor.collect([
```

Comment:
About your slack question: you could add an `doctor.on('warning')` event handler here that prints the message

Reply:
@goto-bus-stop - Sounds good. Thanks!